bin/traffic-splitter.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const program = require('commander')
+const chalk = require('chalk')
+
+const error = chalk.bold.red
+const logErr = (msg) => console.log(error(msg))
+
+program
+  .version('1.0.0')
```

Comment:
You can require the `package.json` file directly and re-use the version from there:
e.g.:
```
const version = require('../pacakge.json').version
```

This will save you the effort of updating the cli version.